### PR TITLE
Test CI/CD develop to main checks

### PR DESCRIPTION
**Pull request by bailliem**
Test to check CICD runs merging to master. 

Want to check that we can merge to master from a clean develop and that documentation automatically builds.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -29,11 +30,11 @@ Suggests:
     vdiffr,
     knitr,
     rmarkdown,
-    survminer,
     forcats,
     glue,
-    here,
-    stringr
+    stringr,
```
Comment (SHAESEN2) [inline DESCRIPTION:35]: if we can avoid this one, please do. It does not have added value as all string manipulations are straightforward with base R

File: DESCRIPTION [hunk 2]
```diff
@@ -29,11 +30,11 @@ Suggests:
...
+    openxlsx,
+    data.table
```
Comment (SHAESEN2) [inline DESCRIPTION:37]: we dont need this. If you get an error because of this, we need to solve it.

File: DESCRIPTION [hunk 3]
```diff
@@ -45,14 +46,10 @@ Imports:
...
     tibble,
     tidyr,
     dplyr,
-    ggpubr,
-    pammtools,
     kableExtra,
-    assertthat,
     gt (>= 0.1.0),
     tidyselect,
     parcats (>= 0.0.1),
```
Comment (SHAESEN2) [inline DESCRIPTION:55]: we dont need this one. as.factor and factor should be sufficient from base R

File: DESCRIPTION [hunk 4]
```diff
@@ -62,7 +59,5 @@ Imports:
     gridExtra,
     plotly,
```
Comment (SHAESEN2) [inline DESCRIPTION:60]: not yet required?

File: R/add_risktable.R [hunk 5]
```diff
@@ -31,35 +31,32 @@ add_risktable <- function(gg, ...){
 #' 
 #' @examples
 #'
+#' library(survival)
```
Comment (SHAESEN2) [inline R/add_risktable.R:34]: remove

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (bailliem) [reply DESCRIPTION:35]: Removed both from the release branch DESCRIPTION file

File: DESCRIPTION (hunk 3 above)
Comment (bailliem) [reply DESCRIPTION:55]: Removed

File: DESCRIPTION (hunk 4 above)
Comment (bailliem) [reply DESCRIPTION:60]: Removed pltly. gridExtra is used in add_annotation

**Pull request closed without merging**